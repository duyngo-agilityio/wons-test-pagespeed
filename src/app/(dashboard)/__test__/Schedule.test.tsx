// Page
import SchedulePage from '../schedule/page';

describe('Schedule page', () => {
  it('should match snapshot', () => {
    const { container } = testLibJestUtils.render(<SchedulePage />);

    expect(container).toMatchSnapshot();
  });
});
