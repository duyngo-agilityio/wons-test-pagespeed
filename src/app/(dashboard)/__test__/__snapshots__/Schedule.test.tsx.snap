// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule page should match snapshot 1`] = `
<div>
  <div
    data-overlay-container="true"
  >
    <script>
      !function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&false)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}else{c.add('light')}if(e==='light'||e==='dark'||!e)d.style.colorScheme=e||'light'}catch(e){}}()
    </script>
    <section>
      <div
        class="base:mb-6 md:mb-7.5 flex items-center justify-between"
      >
        <h2
          aria-level="2"
          class="font-bold font-dm-sans text-blue-800 dark:text-white text-5xl w-full"
        >
          Task Preview
        </h2>
        <button
          aria-expanded="false"
          class="group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 gap-2 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover z-10 aria-expanded:scale-[0.97] aria-expanded:opacity-70 subpixel-antialiased bg-blue-500 border-transparent dark:bg-purple-600 text-white hover:bg-blue-100 px-5 py-3 text-md rounded-[10px] border-[1px] h-auto min-w-max font-medium"
          data-slot="trigger"
          type="button"
        >
          Filter
          <svg
            aria-hidden="true"
            color="#FFFFFF"
            fill="none"
            focusable="false"
            height="13"
            tabindex="-1"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4.853 3.1c0 1.283-1.058 2.324-2.364 2.324C1.184 5.424.125 4.384.125 3.1.125 1.818 1.184.777 2.489.777c1.306 0 2.364 1.04 2.364 2.323Zm7.986-1.017c.572 0 1.036.456 1.036 1.017 0 .562-.464 1.019-1.036 1.019H8.32A1.028 1.028 0 0 1 7.282 3.1c0-.561.464-1.017 1.037-1.017h4.52ZM1.162 9.686h4.52c.572 0 1.037.456 1.037 1.018s-.465 1.019-1.037 1.019h-4.52a1.029 1.029 0 0 1-1.037-1.019c0-.562.464-1.018 1.037-1.018Zm10.349 3.313c1.306 0 2.364-1.04 2.364-2.322 0-1.284-1.058-2.324-2.364-2.324-1.306 0-2.364 1.04-2.364 2.324 0 1.282 1.058 2.322 2.364 2.322Z"
              fill="currentColor"
              fill-rule="evenodd"
            />
          </svg>
        </button>
      </div>
      <div
        class="md:flex md:justify-between md:items-center"
      >
        <div
          aria-label="Custom tabs"
          class="inline-flex text-black dark:text-white p-4 rounded-[3px] px-0 base:w-full md:w-fit"
          data-slot="base"
        >
          <div
            aria-label="Custom tabs"
            aria-orientation="horizontal"
            class="flex h-fit gap-2 items-center flex-nowrap overflow-x-scroll scrollbar-hide bg-white dark:bg-gray-400 shadow-lg border border-gray-200 dark:border-gray-700 p-0 rounded-[5px] base:w-full md:w-fit"
            data-slot="tabList"
            id="react-aria-:r1:"
            role="tablist"
          >
            <button
              aria-selected="true"
              class="z-0 w-full flex group relative justify-center items-center cursor-pointer transition-opacity tap-highlight-transparent data-[disabled=true]:cursor-not-allowed data-[disabled=true]:opacity-30 data-[hover-unselected=true]:opacity-disabled outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 h-8 text-small rounded-small !py-6 font-medium text-gray-700 hover:text-blue-500 dark:text-gray-300 dark:hover:text-white px-4 py-2"
              data-key="list"
              data-selected="true"
              data-slot="tab"
              id="react-aria-:r1:-tab-list"
              role="tab"
              tabindex="0"
              type="button"
            >
              <span
                class="absolute z-0 inset-0 shadow-small bg-blue-500 dark:bg-purple-600 rounded-none"
                data-slot="cursor"
              />
              <div
                class="relative z-10 whitespace-nowrap transition-colors text-blue-800 dark:text-white group-data-[selected=true]:text-white dark:group-data-[selected=true]:text-gray-400"
                data-slot="tabContent"
              >
                List
              </div>
            </button>
            <button
              aria-selected="false"
              class="z-0 w-full flex group relative justify-center items-center cursor-pointer transition-opacity tap-highlight-transparent data-[disabled=true]:cursor-not-allowed data-[disabled=true]:opacity-30 data-[hover-unselected=true]:opacity-disabled outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 h-8 text-small rounded-small !py-6 font-medium text-gray-700 hover:text-blue-500 dark:text-gray-300 dark:hover:text-white px-4 py-2"
              data-key="board"
              data-slot="tab"
              id="react-aria-:r1:-tab-board"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <div
                class="relative z-10 whitespace-nowrap transition-colors text-blue-800 dark:text-white group-data-[selected=true]:text-white dark:group-data-[selected=true]:text-gray-400"
                data-slot="tabContent"
              >
                Board
              </div>
            </button>
            <button
              aria-selected="false"
              class="z-0 w-full flex group relative justify-center items-center cursor-pointer transition-opacity tap-highlight-transparent data-[disabled=true]:cursor-not-allowed data-[disabled=true]:opacity-30 data-[hover-unselected=true]:opacity-disabled outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 h-8 text-small rounded-small !py-6 font-medium text-gray-700 hover:text-blue-500 dark:text-gray-300 dark:hover:text-white px-4 py-2"
              data-key="timeLine"
              data-slot="tab"
              id="react-aria-:r1:-tab-timeLine"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <div
                class="relative z-10 whitespace-nowrap transition-colors text-blue-800 dark:text-white group-data-[selected=true]:text-white dark:group-data-[selected=true]:text-gray-400"
                data-slot="tabContent"
              >
                Timeline
              </div>
            </button>
          </div>
        </div>
        <div
          class="group flex flex-col data-[hidden=true]:hidden w-full relative justify-end data-[has-label=true]:mt-[calc(theme(fontSize.small)_+_10px)] base:w-full md:w-fit"
          data-filled="true"
          data-filled-within="true"
          data-slot="base"
        >
          <div
            class="h-full flex flex-col"
            data-slot="main-wrapper"
          >
            <div
              class="relative w-full inline-flex tap-highlight-transparent flex-row items-center shadow-sm px-3 gap-3 data-[hover=true]:bg-default-200 group-data-[focus=true]:bg-default-100 h-10 min-h-10 rounded-medium transition-background motion-reduce:transition-none !duration-150 outline-none group-data-[focus-visible=true]:z-10 group-data-[focus-visible=true]:ring-2 group-data-[focus-visible=true]:ring-focus group-data-[focus-visible=true]:ring-offset-2 group-data-[focus-visible=true]:ring-offset-background bg-white dark:bg-gray-400 hover:!bg-white dark:hover:!bg-gray-400 focus-within:!bg-white dark:focus-within:!bg-gray-400 group-data-[focus=true]:!bg-white dark:group-data-[focus=true]:!bg-gray-400 pl-5 pr-3.5 py-3 md:max-w-[230px]"
              data-slot="input-wrapper"
              style="cursor: text;"
            >
              <div
                class="inline-flex w-full items-center h-full box-border"
                data-slot="inner-wrapper"
              >
                <input
                  aria-label="Search"
                  class="w-full font-normal bg-transparent !outline-none focus-visible:outline-none data-[has-start-content=true]:ps-1.5 data-[has-end-content=true]:pe-1.5 file:cursor-pointer file:bg-transparent file:border-0 autofill:bg-transparent bg-clip-text text-small group-data-[has-value=true]:text-default-foreground !text-blue-800 dark:!text-white placeholder:text-sm placeholder:text-blue-800/50 dark:placeholder:text-white/50"
                  data-has-end-content="true"
                  data-slot="input"
                  id="react-aria-:r7:"
                  placeholder="Search"
                  type="text"
                  value=""
                />
                <svg
                  class="w-3 h-3 text-blue-800/60 dark:text-white/60 cursor-pointer"
                  fill="currentColor"
                  height="1em"
                  id="search"
                  stroke="currentColor"
                  stroke-width="0"
                  version="1.1"
                  viewBox="0 0 24 24"
                  width="1em"
                  x="0px"
                  xmlns="http://www.w3.org/2000/svg"
                  y="0px"
                >
                  <g>
                    <path
                      d="M20.031,20.79c0.46,0.46,1.17-0.25,0.71-0.7l-3.75-3.76c1.27-1.41,2.04-3.27,2.04-5.31
		c0-4.39-3.57-7.96-7.96-7.96s-7.96,3.57-7.96,7.96c0,4.39,3.57,7.96,7.96,7.96c1.98,0,3.81-0.73,5.21-1.94L20.031,20.79z
		 M4.11,11.02c0-3.84,3.13-6.96,6.96-6.96c3.84,0,6.96,3.12,6.96,6.96c0,3.84-3.12,6.96-6.96,6.96C7.24,17.98,4.11,14.86,4.11,11.02
		z"
                    />
                  </g>
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="w-[275px] bg-white dark:bg-gray-400 p-[20px] rounded-5"
      >
        <div
          class="flex flex-row items-center justify-between mb-[15px]"
        >
          <div
            class="text-md"
          >
            Dashboard Design
          </div>
          <button
            aria-expanded="false"
            aria-haspopup="true"
            class="group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 min-w-20 h-10 text-small gap-2 rounded-medium [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none text-default-foreground data-[hover=true]:opacity-hover z-10 aria-expanded:scale-[0.97] aria-expanded:opacity-70 subpixel-antialiased border-none bg-bone"
            data-slot="trigger"
            data-testid="actions-btn"
            id="react-aria-:rb:"
            type="button"
          >
            <svg
              class="text-blue-800/30 dark:text-white/30"
              fill="currentColor"
              height="14"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
              />
            </svg>
          </button>
        </div>
        <div
          class="relative max-w-fit min-w-min inline-flex items-center justify-between box-border whitespace-nowrap text-small text-default-foreground px-[22px] h-[45px] max-h-[45px] rounded-[22px] bg-blue-500"
        >
          <span
            class="flex-1 font-normal px-2 text-center text-md capitalize text-white dark:text-gray-600"
          >
            medium
          </span>
        </div>
        <div
          class="mt-[20px] text-sm"
        >
          Discussion for management dashboard ui design
        </div>
        <div
          class="mt-[20px]"
        >
          <img
            alt="John Doe"
            data-nimg="1"
            decoding="async"
            height="176"
            loading="lazy"
            src="/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=640&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=256&q=75 1x, /_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=640&q=75 2x"
            style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
            width="235"
          />
        </div>
        <div
          class="mt-[20px]"
        >
          <div
            class="flex items-center"
          >
            <div
              class="relative w-7 h-7 rounded-full "
            >
              <img
                alt="user1"
                class="h-full w-full rounded-full object-cover"
                data-nimg="1"
                decoding="async"
                height="28"
                loading="lazy"
                src="http://localhost/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=32&q=75 1x, /_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75 2x"
                style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                width="28"
              />
            </div>
            <div
              class="relative w-7 h-7 rounded-full -ml-3"
            >
              <img
                alt="user2"
                class="h-full w-full rounded-full object-cover"
                data-nimg="1"
                decoding="async"
                height="28"
                loading="lazy"
                src="http://localhost/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=32&q=75 1x, /_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75 2x"
                style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                width="28"
              />
            </div>
            <div
              class="relative w-7 h-7 rounded-full -ml-3"
            >
              <img
                alt="user4"
                class="h-full w-full rounded-full object-cover"
                data-nimg="1"
                decoding="async"
                height="28"
                loading="lazy"
                src="http://localhost/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=32&q=75 1x, /_next/image?url=https%3A%2F%2Fimg-cdn.pixlr.com%2Fimage-generator%2Fhistory%2F65bb506dcb310754719cf81f%2Fede935de-1138-4f66-8ed7-44bd16efc709%2Fmedium.webp&w=64&q=75 2x"
                style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
                width="28"
              />
            </div>
            <button
              class="z-0 group box-border appearance-none select-none whitespace-nowrap font-normal subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 gap-2 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none bg-default data-[hover=true]:opacity-hover border-transparent dark:text-white px-5 pb-2.5 pt-[11px] text-md rounded-[10px] min-w-max !px-0 relative flex w-7 h-7 items-center justify-center !rounded-full !bg-teal-500 dark:!bg-teal-300 text-white -ml-3 border-2 hover:!bg-teal-600 dark:hover:!bg-teal-400 transition duration-300 ease-in-out"
              type="button"
            >
              <div
                class="flex items-center"
              >
                <svg
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <line
                    x1="12"
                    x2="12"
                    y1="5"
                    y2="19"
                  />
                  <line
                    x1="5"
                    x2="19"
                    y1="12"
                    y2="12"
                  />
                </svg>
              </div>
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
